azure/script.sh

```diff
@@ -0,0 +1,11 @@
+#eval `ssh-agent -s`
```

Comment:
Did you push this file by accident?

Reply:
exactly, that should not be there. I will remove it

Reply:
done